report_thesis/src/sections/background.tex

```diff
@@ -126,3 +126,82 @@ \subsection{Data Normalization}\label{sec:data_normalization}
+In this section, we provide an overview and definitions of the core models used in this work.
+These models form the basis of the final architecture of our proposed pipeline, detailed further in Section~\ref{sec:methodology}.
+
+\subsubsection{PLS}
+First, we provide an overview of \gls{pls}.
+To understand \gls{pls}, it is essential to first understand \gls{pca} and \gls{pcr}.
+
+\gls{pca} is a dimensionality reduction technique that transforms a set of possibly correlated variables into a smaller set of uncorrelated variables called \textit{principal components}.
+First, the data matrix $\mathbf{X}$ is centered by subtracting the mean of each variable to ensure that the data is centered at the origin:
+
+$$
+\mathbf{\bar{X}} = \mathbf{X} - \mathbf{\mu},
+$$
+
+where $\mathbf{\bar{X}}$ is the centered data matrix and $\mathbf{\mu}$ is the mean of each variable.
+
+The covariance matrix of the centered data is then computed:
+
+$$
+\mathbf{C} = \frac{1}{n-1} \mathbf{\bar{X}}^T \mathbf{\bar{X}},
+$$
+
+where $n$ is the number of samples.
+
+Then, the covariance matrix $C$ is decomposed into its eigenvectors $\mathbf{V}$ and eigenvalues $\mathbf{D}$:
+
+$$
+\mathbf{C} = \mathbf{V} \mathbf{D} \mathbf{V}^T,
+$$
+
+where $\mathbf{V}$ is the matrix of eigenvectors and $\mathbf{D}$ is the diagonal matrix of eigenvalues.
+
+The eigenvectors are then ranked by the magnitude of their corresponding eigenvalues in descending order.
+The eigenvectors with the highest eigenvalues are the principal components that capture the most variance in the data.
+Usually, only the top $n$ principal components are retained to reduce the dimensionality of the data, as they are assumed to capture the most significant aspects of the data due to their high variance.
+
+Finally, the original data points are projected onto the space defined by the top $n$ principal components, which transforms $X$ into a lower-dimensional representation:
+
+$$
+\mathbf{X}_{\text{reduced}} = \mathbf{\bar{X}} \mathbf{V}_n,
+$$
+
+where $\mathbf{V}_n$ is the matrix that only contains the top $n$ eigenvectors.
+
+A property of the principal components is that they are uncorrelated, which means that calculating the covariance matrix of each pair of components results in a diagonal matrix where the off-diagonal elements are zero.
+
+\gls{pcr} extends \gls{pca} in the context of regression analysis.
+\gls{pcr} involves performing \gls{pca} on the features $\mathbf{X}$ and then using the principal components with the highest variances as features in a linear regression model.
+This approach can be beneficial when the original features are highly correlated, as the principal components are orthogonal to each other, reducing multicollinearity.
+If $\mathbf{X}$ is decomposed into $\mathbf{TP}^T + \mathbf{E}$ where $\mathbf{T}$ are the scores and $\mathbf{P}$ are the loadings, \gls{pcr} uses $\mathbf{T}$ to predict the target $\mathbf{y}$ as $\mathbf{y} = \mathbf{Tb} + \mathbf{e}$.
```

Comment:
What are $\mathbf{b}$ and $\mathbf{e}$ in the regression equation $\mathbf{y} = \mathbf{Tb} + \mathbf{e}$?
Does $\mathbf{e}$ relate to $\mathbf{E}$?

What is the role of the loadings ($\mathbf{P}$) and scores ($\mathbf{T}$) in PCR? They aren't explained, either - unclear what scores & loadings are.